src/main/java/racingcar/model/Generator.java

```diff
@@ -0,0 +1,39 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+import java.util.List;
+import racingcar.GameSettings;
+
+public class Generator {
+    public List<Car> generateCarsFrom(String names) {
+        List<Car> cars = new ArrayList<>();
+        for (String name : names.split(",")) {
+            cars.add(new Car(name));
+        }
+        return cars;
+    }
+
+    private int generateRandomNumber() {
+        return Randoms.pickNumberInRange(GameSettings.MIN_RANDOM_NUMBER.getNumber()
+                , GameSettings.MAX_RANDOM_NUMBER.getNumber());
+    }
+
+    public List<Integer> generateRandomNumbers(int size) {
+        List<Integer> randomNumbers = new ArrayList<>();
+        for (int i = 0; i < size; i++) {
+            randomNumbers.add(generateRandomNumber());
+        }
+        return randomNumbers;
+    }
+
+    public List<RacingRecord> generateRecordsFrom(String names) {
+        List<RacingRecord> records = new ArrayList<>();
+
+        List<Car> cars = generateCarsFrom(names);
+        for (Car car : cars) {
+            records.add(new RacingRecord(car));
+        }
+        return records;
+    }
```

Comment:
이렇게 list를 생성 및 add해서 반환하는 코드는 stream을 사용하면 add없이 return할 수 있어요!

```suggestion
public List<RacingRecord> generateRecordsFrom(String names) {
    return generateCarsFrom(names).stream()
            .map(RacingRecord::new)
            .collect(Collectors.toList());
}
```

Reply:
2 주차 목표로 `stream 사용`이 있었는데 아직 익숙하지 않아 못했네요 ㅠ
`stream` 사용하는 것이 확실히 코드도 간결하고 가독성 측면에서도 좋은 것 같아요!

이번 주차엔 `stream` 꼭 잘 사용해보겠습니다! 예시 코드까지 작성해주시면서 좋은 피드백 감사해요 👍